src/projects/create-pull-request/create-pull-request.html

```diff
@@ -202,6 +204,7 @@
           this.$.errorToast.show({text: 'Missing pull request description'});
           return;
         }
+        this.loadingPR = true;
```

Comment:
This is not possible the ajax is in pull-request-diff and this method is called with a fire.
I have put it here, so we can stop on error.
Otherwise the spinner wont stop when there is a duplicate project, or such.